plasma/transaction.go

```diff
@@ -21,21 +22,21 @@ type Transaction struct {
 }
 
 type txList struct {
-	BlkNum0           []byte
-	TxIndex0          uint16
-	OIndex0           uint8
-	DepositNonce0     []byte
-	Input0ConfirmSigs [][65]byte
-	BlkNum1           []byte
-	TxIndex1          uint16
-	OIndex1           uint8
-	DepositNonce1     []byte
-	Input1ConfirmSigs [][65]byte
+	BlkNum0           [32]byte
```

Comment:
Would this not just be `uint32` instead of `[32]byte` should encode the same way